lib/install.go

```diff
@@ -119,12 +119,12 @@ func Install(tfversion string, binPath string, mirrorURL string) {
 
 	/* if selected version already exist, */
 	/* proceed to download it from the hashicorp release page */
-	url := mirrorURL + tfversion + "/" + versionPrefix + tfversion + "_" + goos + "_" + goarch + ".zip"
-	zipFile, errDownload := DownloadFromURL(installLocation, url)
+	zipFile, errDownload := DownloadFromURL(installLocation, mirrorURL, tfversion, versionPrefix, goos, goarch)
 
 	/* If unable to download file from url, exit(1) immediately */
 	if errDownload != nil {
-		logger.Fatalf("Error downloading %s: %v", url, errDownload)
+		logger.Fatalf("Error downloading: %v", errDownload)
+		os.Exit(1)
```

Comment:
`logger.Fatalf()` performs `is.Exit(1)`, doesn't it? 🤔 
```suggestion
```

Reply:
I had the same phenomen. It should exit but it doesn't. Hence my manual exit calls.
Maybe I am missing a flag on the logger config or something.
I will have another look. Otherwise I will open an issue with slog